test/srv6-inject/src/nf.rs

```diff
@@ -90,36 +28,27 @@ struct MetaDataz {
 impl Default for MetaDataz {
     fn default() -> MetaDataz {
         MetaDataz {
-            next_header: NextHeader::NoNextHeader,
-            payload_len: 0,
```

Comment:
we're dropping these from here because we no longer need to carry them around, we're just setting them from the ipv6 header when we call swap/insert, yeah? much nicer.

Reply:
yep.